modules/oxide/src/less/theme/components/menubar/promotion.less

```diff
@@ -0,0 +1,33 @@
+//
+// Promotion
+//
+
+@promotion-background-color: @background-color;
+@promotion-background: url("data:image/svg+xml;charset=utf8,%3Csvg height='@{_promotion-height}' viewBox='0 0 40 @{_promotion-height}' width='40' xmlns='http://www.w3.org/2000/svg'%3E%3Crect x='0' y='@{_promotion-divider-height}' width='100' height='1' fill='@{_promotion-row-separator-color}'/%3E%3C/svg%3E") left 0 top 0 @promotion-background-color;
+
+// private variables
+@_promotion-row-separator-color: escape('@{menubar-row-separator-color}');
+@_promotion-height: @promotion-select-height + (@promotion-select-spacing-y * 2) + 1px;
+@_promotion-divider-height: @_promotion-height - 1px;
+@promotion-select-height: @toolbar-button-height;
+@promotion-select-spacing-y: @toolbar-button-spacing-y;
```

Comment:
These don't appear to be used?

Reply:
They are used in the above `promotion-background`-definition.

Reply:
Ahh I had missed that sorry... in that case though then they are using the wrong fallbacks, because this is in the menubar so this should be using `@menubar-select-` not `@toolbar-button-`.

Reply:
Also the last 2 are not private (no underscore prefix) so they should be above the background anyways. Either that or they should be named properly as a private variable.

Reply:
Alright, I'll use the menubar ones.